### ISSUE TITLE
Show tmp dir free space in tmp dir table

### ISSUE THREAD

**Issue by ericaltendorf** [ericaltendorf/plotman#50]
Sure it's  available in other monitoring systems, but it's pretty relevant to tuning plotting.

**Comment by jinthagerman** [comment]
when selecting the tmp drive to plot to, does the remaining space get taken into account?

**Comment by ericaltendorf** [comment]
No, the current logic schedules jobs by max # per temp drive, min time
since last job, and least recently used.
